### PR TITLE
Added DP downloader

### LINKED ISSUE #468: WhatsApp Profile Picture Downloading Script
I would like to add a project which would help us in downloading the profile pictures of all in our contact list. I would like to do in python.
I am a GSSOC'21 participant.

### PR DESCRIPTION
**Pull request by sudipg4112001**
### Issue fixed
Fixes #468 

Please review the PR

### CODE REVIEW

File: WhatsApp_DP_download/readme.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<p align="center"><img src="https://github.com/sudipg4112001/Amazing-Python-Scripts/blob/DP_download/WhatsApp_DP_download/Image.jpg"/></p>
+
+### About this feature:
+
+This is purely a python script to the get profile photos rather download profile photos of all the contacts in our whatsapp contact list.
+- Here we use logger
+- Target the contacts
+- Then, use url of the photos to download the photos.
+
+### This, is a gist of this project..!!
```
Comment (kaustubhgupta) [inline WhatsApp_DP_download/readme.md:10]: This line is not needed

File: WhatsApp_DP_download/profile_pic_download.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# region IMPORTS
```
Comment (kaustubhgupta) [inline WhatsApp_DP_download/profile_pic_download.py:1]: Remove these region comments from the script

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by sudipg4112001** [comment]
I have made the changes. Please review

File: WhatsApp_DP_download/profile_pic_download.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+
```
Comment (kaustubhgupta) [inline WhatsApp_DP_download/profile_pic_download.py:1]: Remove extra spaces from the program

File: WhatsApp_DP_download/readme.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<p align="center"><img src="https://github.com/sudipg4112001/Amazing-Python-Scripts/blob/DP_download/WhatsApp_DP_download/Image.jpg"/></p>
+
+### About this feature:
+
+This is purely a python script to the get profile photos rather download profile photos of all the contacts in our whatsapp contact list.
+
+
+### This, is a gist of this project..!!
```
Comment (kaustubhgupta) [inline WhatsApp_DP_download/readme.md:8]: I said to remove this line: "### This, is a gist of this project..!!" and not the above text. Bring back that text and remove this

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by sudipg4112001** [comment]
I don't think the extra spaces in the program effects the program. Rather, it separates the program into different parts, which indicates different working. But, if you want I can do. But, if it is kept it will be beneficial.

**Comment by sudipg4112001** [comment]
And, I have done the changes in the readme

**Comment by kaustubhgupta** [comment]
> I don't think the extra spaces in the program effects the program. Rather, it separates the program into different parts, which indicates different working. But, if you want I can do. But, if it is kept it will be beneficial.

I also had this same thought process until I read this thread last year, maybe it clears your doubts! https://softwareengineering.stackexchange.com/questions/251216/is-too-much-whitespace-a-bad-thing

**Comment by sudipg4112001** [comment]
Okay.. I will make the required changes. 
I need some time.

**Comment by kaustubhgupta** [comment]
@sudipg4112001 updates?

**Comment by sudipg4112001** [comment]
> @sudipg4112001 updates?

Actually, I am facing some issues with my device. So, I need 2/3 days time.

**Comment by kaustubhgupta** [comment]
> > @sudipg4112001 updates?
> 
> Actually, I am facing some issues with my device. So, I need 2/3 days time.

I can give time for today only

**Comment by kaustubhgupta** [comment]
No response from your side, this PR has been closed

**Pull request closed without merging**